src/www/system_advanced_admin.php

```diff
@@ -624,6 +686,22 @@ function reloadWaitOld () {
                 </td>
               </tr>
               <tr>
+                <td><a id="help_for_webgui_managementaccess" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext('Management Access') ?></td>
+                <td>
+                  <select id="webgui_managementaccess" name="webgui_managementaccess" class="selectpicker">
+                      <option value="interfaces" <?= empty($pconfig['webgui_managementaccess']) || $pconfig['webgui_managementaccess'] === 'interfaces' ? 'selected="selected"' : '';?>>
+                        <?=gettext("Choose Listen Interfaces");?>
+                      </option>
+                      <option value="ip" <?=$pconfig['webgui_managementaccess'] === 'ip' ? 'selected="selected"' : '';?>>
+                        <?=gettext("Specify Management IP");?>
```

Comment:
```suggestion
                        <?=gettext("Listen on specified interfaces (recommended)");?>
                      </option>
                      <option value="ip" <?=$pconfig['webgui_managementaccess'] === 'ip' ? 'selected="selected"' : '';?>>
                        <?=gettext("Listen on specified IP address");?>
```

Usually one configures on the settings page _what the system shall do_. Therefore one might interpret the text as "the system will chose the listen interface".